tbx/project_styleguide/templates/patterns/atoms/motif-heading/motif-heading.html

```diff
@@ -1,5 +1,24 @@
-<h{{ heading_level|default:1 }} class="motif-heading {% if classes %}{{ classes }}{% endif %}" aria-label="{% firstof aria_label heading %}">
-    {% with first_letter=heading|slice:":1" %}
-        <span class="motif-heading__drop-cap {% if first_letter|lower == 'i' %}motif-heading__drop-cap--i{% endif %} {% if first_letter|lower in 'f,p' %}motif-heading__drop-cap--narrow{% endif %} {% if first_letter|lower == 'w' %}motif-heading__drop-cap--narrower{% endif %} {% if first_letter|lower in 't,y' %}motif-heading__drop-cap--narrowest{% endif %}">{{ first_letter }}</span><span>{{ heading|slice:"1:" }}</span>
-    {% endwith %}
-</h{{ heading_level|default:1 }}>
+{% if heading_level > '1' and heading_level < '7' %}
+    {# If the heading level is anything other than one, output the heading tag with the motif styles #}
```

Comment:
Q: On the typography page in the styleguide the `<h2>` for 'Always be curious, daring, and eager to learn' is being output with the `sr-only` class and the heading text in a `<p>` tag - is that correct? From the comment here it sounds like the <p> tag shouldn't be output.


http://localhost:8000/pattern-library/render-pattern/patterns/styleguide/typography/typography.html

Reply:
Looking further into this if you don't put the heading_level numbers in a string it works as the comment describes:

```{% if heading_level > 1 and heading_level < 7 %}```
instead of
```{% if heading_level > '1' and heading_level < '7' %}```

Reply:
that then doesn't work in the template though, as the template thinks the numbers are strings. I think we may need to adjust the yaml in the pattern library instead, where the type is clearly integer again.

Reply:
or I go back to my original solution...

Reply:
I think I'll just go back to my original solution which forced integers to strings in the test.

Reply:
Done - the comparison now works whether it is integers in the pattern library or strings in the template.